server/src/test/java/org/elasticsearch/snapshots/SharedClusterSnapshotRestoreIT.java

```diff
@@ -2483,7 +2483,7 @@ public void testCloseOrDeleteIndexDuringSnapshot() throws Exception {
                 } else {
                     logger.info("--> close index while partial snapshot is running");
                     closedOnPartial = true;
-                    client.admin().indices().prepareClose("test-idx-1").get();
+                    client.admin().indices().prepareClose("test-idx-1").setWaitForActiveShards(ActiveShardCount.DEFAULT).get();
```

Comment:
I don't understand how this helps, because JavaDoc for `setWaitForActiveShards` claims that `ActiveShardCount.DEFAULT` is the default value, so the change should introduce no difference in the behaviour

Reply:
On `7.x` the default is `NONE` https://github.com/elastic/elasticsearch/blob/7.x/server/src/main/java/org/elasticsearch/action/admin/indices/close/CloseIndexRequest.java#L43 in `master` it's `DEFAULT`.